src/components/editor-quill/index.stories.js

```diff
@@ -299,3 +293,57 @@ storiesOf('Editors/EditorQuill', module)
       <Wrapper block={contentBock} />
     )
   })
+  .add('with custom parchment styling', () => {
+    /**
+     * inspired by
+     * - https://stackoverflow.com/a/52851287/1238150
+     * - https://stackoverflow.com/a/38723167/1238150 (complex example)
+     * - https://codepen.io/natterstefan/pen/xmZvxe
+     * - https://codepen.io/natterstefan/pen/zyrxzO
+     */
+    const blockConfig = {
+      registerFormats: Quill => {
+        const Inline = Quill.import('blots/inline')
+        const icons = Quill.import('ui/icons')
+
+        class SpanBlock extends Inline {
```

Comment:
inline classes: is this the intended way to do it? if not, maybe we should show an example which aligns more with "best practices"?

Reply:
yeah, missed that one and gonna change it. Thanks for catching it.